config/processCcd.py

```diff
@@ -1,7 +1,10 @@
 from lsst.meas.algorithms import LoadIndexedReferenceObjectsTask
 from lsst.obs.decam.isr import DecamIsrTask
+from lsst.obs.decam.crosstalk import DecamCrosstalkTask
 config.isr.retarget(DecamIsrTask)
+config.isr.crosstalk.retarget(DecamCrosstalkTask)
```

Comment:
You shouldn't have to retarget in both places.  I think it might be better to not have a separate `isr.py` and just put those overrides here.

Reply:
Hmmm is this because you have two different processCcd-ish configs?  If that's the case, I would like to see the dups removed and just use `isr.py` for everything.

Reply:
Right, it's because there are configs for both processCcd and processCcdCpIsr. Both contain more than just ISR-related config settings, however. The workflow for processing outlined in the obs_decam README uses them. I added the ISR config so the test I wrote which runs ISR only will work.

Reply:
Nevermind, got it. Didn't realize one could load configs from configs... or as Colin put it, import pexInception 😆